crates/cargo-wdk/src/actions/new/mod.rs

```diff
@@ -275,8 +277,8 @@ impl<'a> NewAction<'a> {
     ///   config.toml template content to the destination config.toml file.
     pub fn copy_cargo_config(&self) -> Result<(), NewActionError> {
         debug!("Copying .cargo/config.toml file");
-        create_dir_all(self.cwd.join(".cargo"))?;
-        let cargo_config_path = self.cwd.join(".cargo/config.toml");
+        create_dir_all(self.path.join(".cargo"))?;
+        let cargo_config_path = self.path.join(".cargo/config.toml");
```

Comment:
Similar change as this one - https://github.com/svasista-ms/windows-drivers-rs/pull/7/files#r2099497994